packages/cors-proxy/src/bin/www.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+const cors_proxy = require('cors-anywhere');
+
+const host = process.env.HOST || '0.0.0.0';
+const port = normalizePort(process.env.PORT || 3100);
+cors_proxy.createServer({
+  originWhitelist: ['https://www.recipesage.com', 'https://recipesage.com', 'https://beta.recipesage.com', 'https://api.recipesage.com', 'https://localhost', 'capacitor://localhost', 'http://localhost'],
```

Comment:
I think it'd be useful to add an environment variable where the user could specify an allowed origin for selfhosting purposes. That or disable the whitelist (allow any) for NODE_ENV 'selfhost'.